(chore) replace master/slave with inclusive language (#4459)
tldr-pages/tldr: pages/common/git-subtree.md

- Add a git repository as a subtree:

> `git subtree add --prefix={{path/to/directory/}} --squash {{repository_url}} {{main}}`

Reviewer: ```suggestion
`git subtree add --prefix={{path/to/directory/}} --squash {{repository_url}} {{branch_name}}`
```

- Update subtree repository to its latest commit:

`git subtree pull --prefix={{path/to/directory/}} {{repository_url}} {{main}}`

> 

Reviewer: ```suggestion
`git subtree pull --prefix={{path/to/directory/}} {{repository_url}} {{branch_name}}`
```

> - Merge a subtree repository into main:
> 
> `git subtree merge --prefix={{path/to/directory/}} --squash {{repository_url}} {{main}}`

Reviewer: ```suggestion
- Merge a subtree repository into a specified branch:

`git subtree merge --prefix={{path/to/directory/}} --squash {{repository_url}} {{branch_name}}`
```

- Push commits to a subtree repository:

> `git subtree push --prefix={{path/to/directory/}} {{repository_url}} {{main}}`

Reviewer: ```suggestion
`git subtree push --prefix={{path/to/directory/}} {{repository_url}} {{branch_name}}`
```
